README.md

```diff
@@ -2,24 +2,25 @@
-| /api/v2/energyCal | GET | -                                            | XCalibration                                               | Return calibration coefficients                                           |
-| /api/v2/energyCal | POST | -                                            | -                                                          | Run a energy calibration                                                  |
-| /api/v2/acquisitionParams/user?mode=[mode] | GET | -                                            | ZAcquisitionSettings or XAcquisitionSettings               | Returns user acquisition settings for the given mode                      |
-| /api/v2/acquisitionParams/user?mode=[mode] | PUT | ZAcquisitionSettings or XAcquisitionSettings | -                                                          | Applies user acquisition settings for the given mode                      |
-| /api/v2/acquisitionParams/user?mode=[mode] | POST | -                                            | -                                                          | Reset user acquisition settings for the given mode to factory defaults    |
-| /api/v2/test/{spectra}?mode=[mode]&modelName=[modelName] | POST | ZAcquisitionSettings or XAcquisitionSettings | ZTestResult or XTestResult                                 | Runs a test using the given user acquisition settings                     |
-| /api/v2/acquisitionParams/factory?mode=[mode] | GET | -                                            | ZFactoryAcquisitionSettings or XFactoryAcquisitionSettings | Returns factory acquisition settings for the given mode                   |
-| /api/v2/acquisitionParams/factory?mode=[mode] | PUT | ZFactoryAcquisitionSettings or XFactoryAcquisitionSettings                 | -                                                          | Applies factory acquisition settings for the given mode                   |
-| /api/v2/acquisitionParams/factory?mode=[mode] | POST | -                                            | -                                                          | Reset factory acquisition settings for the given mode to factory defaults |
-| /api/v2/acquire/{spectra}?mode=[mode] | POST | ZFactoryAcquisitionSettings or XFactoryAcquisitionSettings                 | ZAcquisitionResult or XAcquisitionResult                   | Acquire spectra using the given factory acquisition settings              |
-| /api/v2/abort | POST | -                                            | -                                                          | Aborts the currently running operation                                    |
+| URL                                                    | METHOD | BODY                                                       | RESPONSE                                                   | DESCRIPTION                                                               |
```

Comment:
hi @ElenaLoukhnova - is there a reason that you reformatted this table?  The markdown table syntax will take care of lining everything up: https://www.markdownguide.org/extended-syntax/#tables

Reply:
Hmm, it seem IntelliJ IDE made the reformatting, I did not noticed it. My bad, now it is fixed.